packages/front-end/components/Auth/CreateOrganization.tsx

```diff
@@ -126,11 +126,13 @@ export default function CreateOrganization(): ReactElement {
                       </div>
                     </div>
                     <div className={style.recommendedOrgInfo}>
-                      <div className={style.recommendedOrgName}>{org.name}</div>
+                      <div className={style.recommendedOrgName}>
+                        {org?.name}
+                      </div>
                       <div className={style.recommendedOrgMembers}>
-                        {org.members === 1
-                          ? `${org.members} member`
-                          : `${org.members} members`}
+                        {org?.members === 1
+                          ? `${org?.members} member`
+                          : `${org?.members} members`}
```

Comment:
I'm curious about this change. Is it possible there's no organization, and in that case, wouldn't it render `undefined members`? It might make more sense to not render anything that requires an organization, e.g. `{org && <RecommendedOrgSection />}`

Reply:
There's basically no real path to get here (except when I forced it during testing). Thought it would be a bit safer to not have the UI error in this case.

Reply:
Edit again, ended up doing what you suggested basically, conditionally rendering the recommended org flow based on both `org` and `mode`.